testing/integration/otel_test.go

```diff
@@ -887,6 +887,8 @@ func TestOtelFBReceiverE2E(t *testing.T) {
           enabled: true
           paths:
             - {{.InputPath}}
+          prospector.scanner.fingerprint.enabled: false
```

Comment:
just wondering is this a valid `chore` PR if we do changes of configuration in our integration tests? Is this change required to happen because the previous config is no longer valid with the prior version?

Reply:
This is a leftover, it is like this for other tests. Long story short filebeat now has [fingerprint fiile identity by default](https://github.com/elastic/beats/pull/41762), which causes issues in tests that ingest a small file that is not big enough to produce a valid fingerprint. These options are used in most tests because of this.